liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/GenerateFeaturesMojo.java

```diff
@@ -141,6 +147,22 @@ private void generateFeatures() throws PluginExecutionException {
         }
     }
 
+    /**
+     * Gets the binary scanner jar file from the local cache.
+     * Downloads it first from connected repositories such as Maven Central if a newer release is available than the cached version.
+     * Note: Maven updates artifacts daily by default based on the last updated timestamp. Users should use 'mvn -U' to force updates if needed.
+     * 
+     * @return The File object of the binary scanner jar in the local cache.
+     * @throws PluginExecutionException
+     */
+    private File getBinaryScannerJarFromRepository() throws PluginExecutionException {
+        try {
+            return getArtifact(BINARY_SCANNER_MAVEN_GROUP_ID, BINARY_SCANNER_MAVEN_ARTIFACT_ID, BINARY_SCANNER_MAVEN_TYPE, BINARY_SCANNER_MAVEN_VERSION).getFile();
+        } catch (Exception e) {
+            throw new PluginExecutionException("Could not retrieve the binary scanner jar. Ensure you have a connection to Maven Central or another repository that contains the jar", e);
```

Comment:
Is it possible to improve this message because in cases they are not using Maven Central they are likely using a repository they specified in their own pom.xml? That is, say "...or the repository you specified in your project's pom.xml"

Reply:
That was kind of hinted at in "or another repository that contains the jar", but it could be clearer.

How about: "Ensure you have a connection to Maven Central or another repository configured in your pom.xml that contains the binary scanner jar"

Reply:
Sure!